elements/map-menu/map-menu.js

```diff
@@ -234,6 +459,22 @@ class MapMenu extends LitElement {
       }
     });
   }
+
+  __openChanged(e){
+    e.stopImmediatePropagation();
+    e.preventDefault();
+    e.stopPropagation();
+    if(this.isHorizontal){
```

Comment:
test for this.shadowRoot as well just to make me happy. Possible event is set in constructor, fires due to low performing page load and then does not have root yet. Highly unlikely but still

Reply:
A check has been added to the if statement